modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/tag/facet/configuration.jsp

```diff
@@ -78,6 +78,14 @@ TagFacetPortletPreferences tagFacetPortletPreferences = new TagFacetPortletPrefe
 
 				<aui:input label="frequency-threshold" name="<%= PortletPreferencesJspUtil.getInputName(TagFacetPortletPreferences.PREFERENCE_KEY_FREQUENCY_THRESHOLD) %>" value="<%= tagFacetPortletPreferences.getFrequencyThreshold() %>" />
 
+				<aui:select label="order-terms-by" name="<%= PortletPreferencesJspUtil.getInputName(TagFacetPortletPreferences.PREFERENCE_KEY_ORDER) %>" value="<%= tagFacetPortletPreferences.getOrder() %>">
+					<aui:option label="" value="" />
```

Comment:
@felipelorenz Could you remove that blank option and put "term-frequency-descending" as the default option to be displayed?